biologic_mpr.yml

```diff
@@ -0,0 +1,17 @@
+id: >-
+  biologic-mpr
```

Comment:
Minor thing, but I'd prefer `biologic.mpr` as it makes the suffix a little bit more obvious.

Reply:
My only concern here is that these IDs will probably be used for canonical URLs at some point, e.g., `https://registry.marda-alliance.org/filetypes/biologic-mpr` so I guess "." isn't URL safe

Reply:
Noted.